src/containers/WorkspaceView.tsx

```diff
@@ -0,0 +1,50 @@
+import { Button } from '@material-ui/core';
```

Comment:
Here's the new workspace view that takes the place of the old RecordingsView. The import button shows the import instructions in a side panel (with a splitter).